WORKSPACE

```diff
@@ -83,6 +83,29 @@ load("@rules_antlr//antlr:deps.bzl", "antlr_dependencies")
 antlr_dependencies()
 
 
+##################################
+# Load Distribution dependencies #
+##################################
+
+# TODO: rename the macro we load here to deploy_github_dependencies
+load("@graknlabs_bazel_distribution//github:dependencies.bzl", "github_dependencies_for_deployment")
+github_dependencies_for_deployment()
+
+load("@graknlabs_build_tools//bazel:dependencies.bzl", "bazel_rules_docker")
+bazel_rules_docker()
+
+load("@io_bazel_rules_docker//repositories:repositories.bzl",
+bazel_rules_docker_repositories = "repositories")
+bazel_rules_docker_repositories()
```

Comment:
why do we need to load all these docker dependencies, @lolski?
- `bazel_rules_docker()`
- `bazel_rules_docker_repositories()`
- `container_pull()`

Is there anything in `@graknlabs_graql` that requires Docker?

Reply:
good point, I thought they were transitive dependencies but turns out they're not.

Reply:
fixed in 8d1dfa2a9055b3f9ed07e7ffe7c94c8421711400